src/intern.rs

```diff
@@ -182,3 +182,12 @@ impl<T: Hash + Eq> Index<Token> for Interner<T> {
         &self.tokens[index.0 as usize]
     }
 }
+
+// TODO: remove in favor of BuildHasher::hash_one once compilers older than 1.71
```

Comment:
I am fine with bunping msrv for this, I loosly follow firefox MSRV and that is on 1.76 at this point.